firefox-ios/firefox-ios-tests/Tests/XCUITests/TrackingProtectionTests.swift

```diff
@@ -88,6 +88,8 @@ class TrackingProtectionTests: BaseTestCase {
         navigator.openURL(differentWebsite)
         waitUntilPageLoad()
         mozWaitForElementToExist(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection])
+        XCTAssert(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection].isHittable)
+        sleep(2)
```

Comment:
This would affect all test executions right? then I would remove this.. if we need this specifically for iOS 15, then I would add an if statement